chromium_src/chrome/browser/resources/tab_search/auto_tab_groups/auto_tab_groups_page.html.ts

```diff
@@ -62,6 +62,51 @@ function getTopicsHtml(this:  AutoTabGroupsPageElement) {
   `)
 }
 
+export function getEnableTabFocusHtml(this:  AutoTabGroupsPageElement) {
+  return html`
+    <div class="enable-tab-focus-wrapper">
+      <div class="enable-tab-focus-header">
+        ${this.showBackButton ? html`
+          <div>
+            <leo-button
+              size="medium"
+              kind="plain-faint"
+              fab
+              @click=${this.onBackClick_}
+            >
+              <leo-icon name="arrow-left"></leo-icon>
+            </leo-button>
+          </div>
+        ` : ''}
```

Comment:
Maybe the header here should be extracted to its own function? Seems almost identical to the default header

Reply:
Extracted the `header` to it's own function!